### PR TITLE
Add option to reindex every website (bug 1161871)

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: lib/es/management/commands/reindex.py [hunk 1]
```diff
@@ -194,7 +195,11 @@ def handle(self, *args, **kwargs):
 
         if index_choice:
             # If we only want to reindex a subset of indexes.
-            INDEXES = INDEX_DICT.get(index_choice, INDEXES)
+            INDEXES = INDEX_DICT.get(index_choice, None)
```
Comment (mstriemer) [inline lib/es/management/commands/reindex.py]: Assigning this to `INDEXES` worries me but I guess it works.

Comment (mstriemer) [inline lib/es/management/commands/reindex.py]: What if you want to index everything? This removes support for that?

Comment (mstriemer) [inline lib/es/management/commands/reindex.py]: Nevermind, only if you provide an index. Makes sense to me!

**Comment by mstriemer** [comment]
Not crazy about reassigning `INDEXES` would rather see `indexes = INDEXES` instead of `global INDEXES` but since this is a management command I guess it doesn't matter and it's clearly been working.

r+

**Comment by diox** [comment]
Not crazy about the code either, but I didn't want to start a refactor just to add this.

**Comment by robhudson** [comment]
I added this also in a branch I have so :+1:

**Pull request merged**